src/impls.rs

```
@@ -539,6 +539,22 @@ mod bstr {
         }
     }
 
+    #[cfg(fetature = "std")]
```

This is silently passing, I think, because of the typo.

---

🤦

---

The fetature is dead 🦀